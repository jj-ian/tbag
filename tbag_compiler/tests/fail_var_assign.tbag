/* Authors: Maria */
true {
        testvar();
        endgame;
}

func int testvar() {
        int [10] i;
        i = 3; 
        return 0;
}
