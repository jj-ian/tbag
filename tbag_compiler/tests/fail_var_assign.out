Fatal error: exception Failure("Left hand side of assignment statement must
                be a non-array variable")
